### PR TITLE
DROOLS-2332: Increase coverage of column rendering strategy

### PR DESCRIPTION
**Pull request by jomarko**
Dependent on #195 

@manstis  please have a look, I will rebase once your PR is in.

### CODE REVIEW

File: uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/columns/impl/BoundingBoxPathClipperFactory.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package org.uberfire.ext.wires.core.grids.client.widget.grid.renderers.columns.impl;
```
Comment (manstis) [inline uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/columns/impl/BoundingBoxPathClipperFactory.java]: License please.

File: uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/test/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/columns/impl/ColumnRenderingStrategyFlattenedTest.java [hunk 2]
```diff
@@ -0,0 +1,212 @@
+package org.uberfire.ext.wires.core.grids.client.widget.grid.renderers.columns.impl;
```
Comment (manstis) [inline uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/test/java/org/uberfire/ext/wires/core/grids/client/widget/grid/renderers/columns/impl/ColumnRenderingStrategyFlattenedTest.java]: License please.

**Comment by jomarko** [comment]
@manstis I rebased so PR contains just changes from my commit, however, I am not sure while also your original commits are listed, would you mind re-review please?

**Comment by manstis** [comment]
@jomarko IDK why either but this PR only contains your changed files.. I'll squash the commits and remove the spurious commit messages when this goes in!

**Comment by jomarko** [comment]
@manstis ready for merge I think.

**Pull request merged**